src/agent/src/rpc.rs

```diff
@@ -1412,15 +1394,21 @@ pub fn start(s: Arc<Mutex<Sandbox>>, server_address: &str) -> TtrpcServer {
         Box::new(HealthService {}) as Box<dyn protocols::health_ttrpc::Health + Send + Sync>;
     let health_worker = Arc::new(health_service);
 
+    let image_service = Box::new(image_rpc::ImageService::new(s))
+        as Box<dyn protocols::image_ttrpc::Image + Send + Sync>;
+
     let aservice = protocols::agent_ttrpc::create_agent_service(agent_worker);
 
     let hservice = protocols::health_ttrpc::create_health(health_worker);
 
+    let iservice = protocols::image_ttrpc::create_image(Arc::new(image_service));
```

Comment:
Nit: I know this is consistent with the code that was already there, but I'd consider making this clearer by creating an `agent_service`, `health_service` and `image_service`.

Reply:
> Your means like this?
> 
> ```
> let agent_service = protocols::agent_ttrpc::create_agent_service(agent_worker);
> let health_service = protocols::health_ttrpc::create_health(health_worker);
> let image_service = protocols::image_ttrpc::create_image(Arc::new(image_service));
> ```

Yes, I believe this is what @jodh-intel means.

Reply:
Yes, exactly that. Not essential, but I think it help the clarity a little.

Reply:
fixed